src/python/pythonUtils.ts

```diff
@@ -28,6 +28,26 @@ export function getEnvInt(key: string): number | undefined {
   return isNaN(parsed) ? undefined : parsed;
 }
 
+/**
+ * Resolves the Python executable path from explicit config and environment.
+ * This centralizes the fallback logic: configPath > PROMPTFOO_PYTHON env var.
+ *
+ * Note: Does NOT apply the final 'python' default - that's handled by
+ * validatePythonPath. This preserves the distinction between "explicitly
```

Comment:
a thought- should this be part of the logic in validatePythonPath? instead of having validation / assignment in 2 places